### ISSUE TITLE
Default to Symfony Serializer's name converter

### ISSUE THREAD

**Issue by sroze** [api-platform/core#1159]
When using Symfony Framework bundle, we can configure the Serializer's name converter using for example the following configuration:
```yaml
framework:
    serializer:
        name_converter: serializer.name_converter.camel_case_to_snake_case
```

Such configuration will not be picked up by ApiPlatform and in such example, I'd need to configure the `api_platform.name_converter` configuration as well. What if we, by default, use the framework configured in Symfony's Framework Bundle?

**Comment by meyerbaptiste** [comment]
Can be closed (see #1202).

**Comment by Simperfit** [comment]
Thanks @meyerbaptiste
